packages/cli/api-importers/openapi/openapi-ir-to-fern-tests/src/__test__/__snapshots__/openapi/merge.json

```diff
@@ -2279,80 +2214,6 @@ Fetch from the `LIST Employments` endpoint and filter by `ID` to show all employ
               "openapi": "../openapi.yml",
             },
           },
-          "EndUserDetailsRequest": {
```

Comment:
@amckinney these would cause breaks for merge

Reply:
Discussed offline - this is actually fixing a bug for Merge, so we can communicate that change to them after the CLI is released.